src/mpl/include/mpl_trmem.h

```diff
@@ -230,6 +230,89 @@ typedef struct {
   Utility
   M*/
 
+#ifdef HAVE_MM256_STREAM_SI256
+#include <immintrin.h>
+#endif
+#include "string.h"
+
+static inline void MPL_Memcpy_stream(void *dest, const void *src, size_t n)
+{
+#ifdef HAVE_MM256_STREAM_SI256
+    if (n <= 256) {
+        memcpy(dest, src, n);
+        return;
+    }
+
+    char *d = (char *) dest;
+    const char *s = (const char *) src;
+
+    /* Copy the first 63 bits or less (if the address isn't 64-bit aligned) using a regular memcpy to
+     * make the rest faster */
```

Comment:
"bits" is not "bytes". I think there are inconsistencies here.

Reply:
I'm not sure what's inconsistent. I meant bits throughout.